akka-actor-typed-tests/src/test/scala/akka/actor/typed/internal/adpater/PropsAdapterSpec.scala

```diff
@@ -18,7 +19,37 @@ class PropsAdapterSpec extends AnyWordSpec with Matchers {
     "default to akka.dispatch.SingleConsumerOnlyUnboundedMailbox" in {
       val props: Props = Props.empty
       val pa: actor.Props = PropsAdapter(() => Behaviors.empty, props, rethrowTypedFailure = false)
-      pa.mailbox shouldEqual "akka.actor.typed.default-mailbox"
+      pa.mailbox === ("akka.actor.typed.default-mailbox")
+
+      val props2: Props = Props.empty.withNext(MailboxSelector.defaultMailbox())
```

Comment:
`withNext` is internal api, can't this test be written with the public: `props2 = MailboxSelector.defaultMailbox()`

similar below...